[UPDATE] Updated 'About Docs' and added 'Navbar' page in Docs Section

himanshu-03 opened the pull request:
- Updated  `About Docs` page in Docs Section
- Changed the label of `Docs Site` to `Docs` in `sidebar.js` file

vish-rt reviewed (approved):
Looks good to me

demonkillerr commented on sidebars.js:
```diff
@@ -37,7 +37,7 @@ const sidebars =
         {
           type: 'category',
-          label: 'Docs Site',
+          label: 'Docs',
```
We need to specify what this documentation is about. Calling it **'docs'** is very counter-intuitive.

himanshu-03 replied:
Changes updated!

demonkillerr reviewed (changes requested):
Rename **Docs** to something else, or keep it that way

himanshu-03 commented:
- Updated  `About Docs` page in Docs Section
- Restored the label of `Docs` to `Docs Site` in `sidebar.js` file
- Added and updated 'Navbar' page in Docs Section

demonkillerr reviewed (approved):
looks good
